docs/install.md

```diff
@@ -0,0 +1,31 @@
+Installing the Pelican Platform
+===============================
+
+This document explains how a user can download and install the Pelican client. 
+
+Installation Steps:
+--------------------
+
+### Install the Pelican Platform Binary
+Navigate to the [Pelican release page](https://github.com/PelicanPlatform/pelican/releases/). Download the proper binary for the system you are running on and select which version you would like to install. 
```

Comment:
I think we should provide a little more information here so that people who are less tech-savvy can figure out which platform they should download on their own. For example, it might not be immediately clear to people whether they should be downloading an RPM, a DEB or an APK file. For the .deb files, we call the x86_64 builds "amd64" which might also be unclear to people. For Mac folks, I wouldn't expect the average person to know that they should grab the Darwin builds. Can you perhaps make a table that people can use to pick the right download?